war/src/main/java/io/lumeer/engine/controller/DatabaseInitializer.java

```diff
@@ -40,6 +45,11 @@
    @Inject
    private DataStorageDialect dataStorageDialect;
 
+
+   public void init(@Observes @Initialized(RequestScoped.class) Object init) {
```

Comment:
It can't be ApplicationScope, because it doesn't work this way (And I don't know why). This method ensure that `DatabaseInitializer` class is initialized so i put code into `PostConstruct` because I saw one solution programmed like this..